src/configs/git_commit.rs

```diff
@@ -0,0 +1,25 @@
+use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
+
+use ansi_term::{Color, Style};
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig)]
+pub struct GitCommitConfig<'a> {
+    pub style: Style,
```

Comment:
Total nitpick.
Could we please reorder this to have unique configuration at the top, to maintain consistency with other module configuration. 😄
- hash
- prefix
- suffix
- style
- disabled